src/Tasks/Microsoft.NET.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -91,7 +91,7 @@ protected override void ExecuteCore()
 
             if (!string.IsNullOrEmpty(RollForward))
             {
-                if (!RollForwardValues.Any(v => string.Equals(RollForward, v, StringComparison.OrdinalIgnoreCase)))
+                if (!RollForwardValues.Contains(RollForward, StringComparer.OrdinalIgnoreCase))
```

Comment:
Why did you decide to change from any to contains here?

Reply:
They're both similar, but for large collections the underlying structure can impact performance I think. `Any` has to enumerate all the elements, but `Contains` can be faster, e.g. a list vs. hash set. I'm good with the switching to the `Contains ` here.

Reply:
I'm trying to remember this since PR is old. `RollForwardValues` is actually just an array `string[]`, so it's still resolving to a LINQ method. 

`Contains`/`Any` have no functional perf differences, so I think I just [wanted to avoid paying unconditionally for the lambda closure allocations](https://github.com/dotnet/roslyn/issues/20777). Obviously it won't be a huge benefit in the big picture, but it was easy to cleanup.